roles/percona_xtradb_cluster/tasks/main.yml

```diff
@@ -47,3 +47,15 @@
     wait_condition:
       type: ready
```

Comment:
this test is more checking where mysql is not ready for keycloak db careate, as it should be ready by this wait condition